installer/installer.py

```diff
@@ -36,8 +36,10 @@
 ###############################################################
 ###     				Cosmovisor Config      				###
 ###############################################################
-COSMOVISOR_BINARY_URL = "https://github.com/cosmos/cosmos-sdk/releases/download/cosmovisor%2Fv1.1.0/cosmovisor-v1.1.0-linux-amd64.tar.gz"
+DEFAULT_LATEST_COSMOVISOR_VERSION = "v1.3.0"
+COSMOVISOR_BINARY_URL = f"https://github.com/cosmos/cosmos-sdk/releases/download/cosmovisor%2F{DEFAULT_LATEST_COSMOVISOR_VERSION}/cosmovisor-{DEFAULT_LATEST_COSMOVISOR_VERSION}-linux-amd64.tar.gz"
```

Comment:
Just realised that this (and our previous version) would _always_ download the linux_amd64 version of Cosmovisor. (`x86_64` is what we call our `amd64` build.) At a later step in the process, we determine which platform architecture the installer is running and accordingly download the appropriate binary.

We should do something similar here. If the user is on a Linux amd64/x86_64 systems, we should download the `linux_amd64` binary. Otherwise, when the user is running Linux ARM64, we should download the appropriate binary for Cosmvisor.

Note that Cosmovisor also generates assets for MacOS/Darwin, Windows etc - we don't need to account for these, since cheqd-node is not compiled for these platforms.